app/model.js

```diff
@@ -11,6 +11,37 @@ const fetchTopics = () =>
     })
 }
 
+
+const fetchComments = (params) =>
+{
+    if(/[0-9]+/g.test(params))
+    {
+        return db.query
+        (
+            `
+            SELECT *
+            FROM comments
+            WHERE article_id = ${params};
+            `
+        )
+        .then((response) =>
+        {
+            if(response.rowCount !== 0)
+            {
+                return response.rows
+            }
+            else
+            {
+                return Promise.reject({status: 404, msg: 'invalid article ID: ID not found'})
```

Comment:
This is one more thing that needs to be done. There could be a situation where an article could have no comments? So if your rowCount is 0, we need to do a check for whether the article actually exists or not because in both cases, an empty array would be returned. 

If the article exists - 200 status could and an empty array is fine to return 
If the article doesn't exist - 404 status code and error message

Please add this and test it, then I am happy for you to merge